src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/DefaultJsonTypeInfoResolver.cs

```diff
@@ -35,7 +38,13 @@ private DefaultJsonTypeInfoResolver(bool mutable)
             s_defaultSimpleConverters ??= GetDefaultSimpleConverters();
         }
 
-        /// <inheritdoc/>
+        /// <summary>
+        /// Resolves a reflection-derived JSON contract for a given <paramref name="type"/> and <paramref name="options"/> configuration.
```

Comment:
This is virtual.  Is this guaranteed?

Reply:
"this" == "reflection-derived"

I suggest the summary should just say `Resolves a JSON contract for a given <paramref name="type"/> and <paramref name="options"/> configuration.`  A `<remarks>...</remarks>` section can further provide commentary on the behavior of the base implementation, noting that a derived implementation may do something completely different.